ferrischat_webserver/src/invites/get_guild_invites.rs

```diff
@@ -0,0 +1,71 @@
+use actix_web::{HttpRequest, HttpResponse, Responder};
+use ferrischat_common::types::{InternalServerErrorJson, Invite, NotFoundJson};
+use ferrischat_macros::get_db_or_fail;
+
+/// GET api/v0/guilds/{guild_id}/invites
+pub async fn get_guild_invites(req: HttpRequest, auth: crate::Authorization) -> impl Responder {
+    let db = get_db_or_fail!();
+    let guild_id = get_item_id!(req, "guild_id");
+    let bigint_guild_id = u128_to_bigdecimal!(guild_id);
+
+    let member_query = sqlx::query!(
+        "SELECT user_id FROM members WHERE user_id = $1, guild_id = $2",
```

Comment:
here you are fetching the user id, with a user id. do you mean to fetch the member id?

Reply:
That syntax I put is wrong (need AND, not comma), but no I just need to make sure the user is in the guild, and if he is, then off the return value

Reply:
I just need to see if they're in the guild or not (if its not None, they are), so i dont care what it returns.

Reply:
although, as someone pointed out (and then deleted), i need to fix that syntax from `WHERE user_id = $1, guild_id = $2` to `WHERE user_id = $1 AND guild_id = $2`

Reply:
yeah agreed. You might want to also get member_id from the table since that is more coherent with the code you have written. It should still work as you intended.

Reply:
members have a separate member id? i thought it was just a user object connected with guild_id and other guild attributes like roles etc.

Reply:
mayebe i should just do SELECT *

Reply:
I'll change my query to `SELECT * FROM members WHERE user_id = $1 AND guild_id = $2`

Reply:
oh wait.

sorry, what you have is fine. there is no member ID